File: front/src/models/Card.ts
export class Card {

    constructor(){
        this.id = 0;
        this.link = '';
        this.description = '';
        this.createdAt = new Date();
    }

    id!: number;
    link!: string;
    description!: string;
    createdAt!: Date | string;
}

File: back/codesApi/Models/Card.cs
namespace codesApi.Models
{
    public class Card
    {
        public Card() { }
        public Card(int id, string link, DateTime createdAt, string description)
        {
            Id = id;
            Link = link;
            Description = description;
            CreatedAt = createdAt;

        }
        public int Id { get; set; }
        public string Link { get; set; }
        public DateTime CreatedAt { get; set; }
        public string Description { get; set; }
    }
}